.github/workflows/build.yml

```
@@ -251,3 +251,6 @@ jobs:
           tags: |
             docker.io/hashicorp/${{env.repo}}:${{env.version}}
             public.ecr.aws/hashicorp/${{env.repo}}:${{env.version}}
+          dev_tags: |
+            hashicorppreview/${{ env.repo }}:${{ env.version }}-dev
+            docker.io/hashicorppreview/${{ env.repo }}:${{ env.version }}-${{ github.sha }}
```

nit: it looks like the convention in the rest of this file is to not use spaces between the brackets.
```suggestion
            hashicorppreview/${{env.repo}}:${{env.version}}-dev
            docker.io/hashicorppreview/${{env.repo}}:${{env.version}}-${{github.sha}}
```

---

Should these tags be different? It looks like one is:
```
v1.12.0-dev
```
And the other is
```
v1.12.0-aabbcc
```

---

(the docker.io part is optional)

but the difference between the two tags is one updates `<version>-dev` to be the latest commit on that release branch whereas the other one will be pushed per commit

---

Got it. I guess it's small but it would be nice if the two URIs are consistent (either both include `docker.io` domain or both omit it), but not that big of a deal :/